src/style.css

```
@@ -0,0 +1,80 @@
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+html,
+input {
+  font-family: "Avenir", Helvetica, sans-serif;
+}
+
```

```suggestion
html {
  font-family: "Avenir", Helvetica, sans-serif;
}

```
you can inherit fonts in classes where it needed